### ISSUE TITLE
Admin UI: Multi-select dropdowns don't show any options

### ISSUE THREAD

**Issue by intmainvoid** [exogee-technology/graphweaver#242]
**Describe the bug**
No valid records are shown in the dropdown on existing records that relate to multiple records.

**To Reproduce**

1. Create a new Postgres database populating the DB with the schema with the supplied DDL (see below)
2. Add some sample records to the 3 tables
3. Create a new project using the command: `npx graphweaver@latest init`
4. Follow the prompts to create a new GW project
5. Change current directory to the project directory
6. Run the command: `npx graphweaver@latest import postgresql` to kick off the Postgres importer
7. Follow the prompts to import the specified database
8. Wait until it completes
9. Start Graphweaver by running `pnpm start`
10. Navigate to `http://localhost:9000/` using the browser
11. Click on the `Voyage` entity to display the Voyage records (which you have hopefully added in the step above)
12. Click on one of the voyage records that have the `exporter_voyages` column populated. This will open the side panel
13. Observe that - in the field `exporter-voyages` - there is "1 Selected" (or however many there are selected).
14. Click on the dropdown list and observe that the list only has a single empty record here. Additionally, when removing the "n Selected" lozenge by clicking on 'x', then selecting the dropdown. There is nothing in the list.

**Expected behavior**
When clicking on the multi-select field in the detail side panel in the Admin UI for any given record, the possible relatable records are displayed.

**Actual behavior**
Blanks or placeholder records are displayed when clicking on the dropdown list.

**Screenshots or Screencasts**
https://github.com/exogee-technology/graphweaver/assets/719151/ca36deca-4df2-4ca5-b182-9e7aba4ba97c
![image](https://github.com/exogee-technology/graphweaver/assets/719151/be687d70-052d-412b-a350-3494d3e31b24)
![image](https://github.com/exogee-technology/graphweaver/assets/719151/27fdc58c-84b0-475e-8feb-c743fdc370b8)

**Desktop (please complete the following information):**

- OS and version: MacOS 13.5.2 (22G91)
- Postgres version: Version 2.6.8a (228)
- Node: v20.10.0
- pnpm: 8.10.5

**Additional context**
DDL to create the database:
```
CREATE TABLE voyage (
    id BIGSERIAL PRIMARY KEY,
    name character varying(255) NOT NULL
);

CREATE TABLE exporter (
    id BIGSERIAL PRIMARY KEY,
    company character varying(255) NOT NULL
);

CREATE TABLE exporter_voyages (
    id BIGSERIAL PRIMARY KEY,
    exporter_id bigint,
    voyage_id bigint REFERENCES voyage(id) ON DELETE CASCADE ON UPDATE CASCADE
);
```

**Comment by agaw** [comment]
Thanks @intmainvoid, I've run through the steps provided but I cannot recreate the issue. I suspect this is because this PR has now part of the latest release: https://github.com/exogee-technology/graphweaver/pull/244. There were some related issues fixed in https://github.com/exogee-technology/graphweaver/pull/255 that will be available in the next release too. I'll leave this issue open for now and retest once the next version is public.

**Comment by agaw** [comment]
Confirmed this is now fixed in version 0.2.13.

**Comment by agaw** [comment]
Confirmed this is now fixed in 0.2.13.
